Error: @openzim/libzim@2.4.2 install

info lifecycle @openzim/libzim@2.4.2~install: Failed to exec install script
Error: @openzim/libzim@2.4.2 install: `npm run download && node-gyp rebuild -v && npm run bundle`
tried installing on node 10, 11 and 12
also tried it on windows and linux
![image](https://user-images.githubusercontent.com/17952364/93723275-8d765080-fb9d-11ea-8824-96d86edcd9c4.png)

tried to install it on debian 10, also have tried to run it on a wsl debian distro (OS: Debian GNU/Linux 10 (buster) on Windows 10 x86_64)

```
npm WARN notsup Unsupported engine for wikimedia-page-library@6.5.0: wanted: {"node":"10.15.3","npm":"6.4.1"} (current: {"node":"14.8.0","npm":"6.14.7"})
npm WARN notsup Not compatible with your version of node/npm: wikimedia-page-library@6.5.0

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @openzim/libzim@2.4.2 install: `npm run download && node-gyp rebuild -v && npm run bundle`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @openzim/libzim@2.4.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Venipa\AppData\Roaming\npm-cache\_logs\2020-09-21T07_39_47_969Z-debug.log```

Same issue experienced on Ubuntu 20.04, using Node 11.14.0 via nvm.

I then installed make and build-essential via apt and it got further but now fails at gifsicle. See below.

`(node:7242) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added. Use emitter.setMaxListeners() to increase limit
  ⚠ Response code 404 (Not Found)
  ⚠ gifsicle pre-build test failed
  ℹ compiling from source
  ✖ Error: Command failed: /bin/sh -c autoreconf -ivf
/bin/sh: 1: autoreconf: not found
...
...
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gifsicle@5.1.0 postinstall: `node lib/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gifsicle@5.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/aaronduce/.npm/_logs/2020-11-09T11_55_08_091Z-debug.log
`

--- a/README.md
+++ b/README.md
@@ -35,10 +35,12 @@ Run `mwoffliner --help` to get all the possible options.
 ## Prerequisites
 
 - *NIX Operating System (GNU/Linux, macOS, ...)
-- [NodeJS](https://nodejs.org/en/)
 - [Redis](https://redis.io/)
+- [NodeJS](https://nodejs.org/en/) version 10 or greater
 - [Libzim](https://github.com/openzim/libzim) (On GNU/Linux & macOS we automatically download it)
-- Various build tools that are probably already installed on your machine (libjpeg, gcc)
+- Various build tools that are probably already installed on your
+  machine (packages `libjpeg-dev`, `autoconf`, `automake`, `gcc` on
+  Debian/Ubuntu)
 
 ... and an online Mediawiki with its API available.
 
@@ -129,6 +131,23 @@ Install Redis:
 ```bash
 sudo apt-get install redis-server
 ```
+
+## Troubleshooting
+
+Older GNU/Linux distributions and/or versions of Node.js might be
+shipped with a deprecated version of `npm`. Older versions of `npm`
+have incompatbilities with certain versions of Node.js and might
+simply fail to install `mwoffliner` package.
+
+We recommend to use a recent version of `npm`. Recent versions can
+perfectly deal with older Node.js 10. Do install the packaged
+version of `npm` and then use it to install a newer version like:
+
+```bash
+sudo npm install --unsafe-perm -g npm
+```
+
+Don't forget to remove the packaged version of `npm` afterward.
 
 License
 -------

